--- a/debian/control
+++ b/debian/control
@@ -15,6 +15,7 @@
                golang-github-coreos-vcontext-dev,
                golang-github-google-renameio-dev,
                golang-github-google-uuid-dev,
+               golang-github-opencontainers-selinux-dev,
                golang-github-pin-tftp-dev,
                golang-github-spf13-pflag-dev (>= 1.0.6~git20210604),
                golang-github-stretchr-testify-dev,
